doc/reference/guidelines.rst

```diff
@@ -141,27 +181,23 @@ Security, View and Action
 
 Use the following pattern :
 
-* For a menu : *<model_name>_menu*
+* For a menu : *<model_name>_menu_<action>* where *action* is a little explanation of the action (use *root* for the main menu).
 * For a view : *<model_name>_view_<view_type>*, where *view_type* is kanban, form, tree, search, ...
```

Comment:
Isn't `_view_` redundant here? `<model_name>_<view_type>` would be expressive enough.